src/main/java/com/flippingutilities/utilities/ListUtils.java

```diff
@@ -1,7 +1,10 @@
 package com.flippingutilities.utilities;
 
-import java.util.ArrayList;
-import java.util.List;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+import java.util.*;
+import java.util.function.Function;
 import java.util.function.Predicate;
 import java.util.stream.IntStream;
```

Comment:
won't `util.*` include the other utils regardless, like Function/Predicate/IntStream, making them irrelevant?

Reply:
I tried to delete the more specific imports but it complained lol...anyway, I removed the star import and made em all specific